regression/goto-harness/havoc-global-int-02/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
:no_entry_sign: As a rule - tests marked as knownbug should reference either a jira ticket or github issue that explains how they should be turned back on - otherwise we might as well delete them. Could you raise a Jira ticket to this effect and put it in the comment section of each desc file